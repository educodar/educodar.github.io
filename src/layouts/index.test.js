import React from "react"
import renderer from "react-test-renderer"
import Layout from "."

describe("Default layout", () => {
  beforeAll(() => {
    jest.mock("../hooks/useSiteMetadata")
  })

  /**
   * @todo Fix TypeError on layout test: Cannot read property 'title' of undefined
   */
  xit("renders correctly", () => {
    const tree = renderer.create(<Layout>Content</Layout>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
