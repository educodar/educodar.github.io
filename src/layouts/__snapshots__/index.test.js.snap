// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default layout renders correctly 1`] = `
Array [
  <header
    className="MuiBox-root MuiBox-root-44 MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    itemScope={true}
    itemType="https://schema.org/WPHeader"
    role="banner"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      <h1
        className="MuiTypography-root MuiTypography-body1"
      >
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
          href="/"
          onBlur={[Function]}
          onFocus={[Function]}
          underlineHover={true}
        >
          <strong>
            Educodar
          </strong>
        </a>
      </h1>
    </div>
  </header>,
  <hr
    className="MuiDivider-root"
  />,
  <main
    className="MuiBox-root MuiBox-root-92"
    role="main"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthMd"
    />
  </main>,
  <hr
    className="MuiDivider-root"
  />,
  <footer
    className="MuiBox-root MuiBox-root-101"
    itemScope={true}
    itemType="https://schema.org/WPFooter"
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthMd"
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph"
      >
        2019-
        2020
         
        <strong>
          Educodar
        </strong>
      </p>
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
        href="/codigo-de-conduta"
        onBlur={[Function]}
        onFocus={[Function]}
        rel={null}
        target="_self"
      >
        Código de Conduta
      </a>
    </div>
  </footer>,
]
`;
